thirdparty/kpvcrlib/CMakeLists.txt

```diff
@@ -70,20 +73,23 @@ if(DEFINED ENV{WIN32})
     find_library(PNG_LIB NAMES "png16-16" PATHS $ENV{LIBS_DIR})
     find_library(FREETYPE_LIB NAMES "freetype-6" PATHS $ENV{LIBS_DIR})
     find_library(HARFBUZZ_LIB NAMES "harfbuzz-0" PATHS $ENV{LIBS_DIR})
+    find_library(UTF8PROC_LIB NAMES "harfbuzz-2" PATHS $ENV{LIBS_DIR})
```

Comment:
harfbuzz-2?

Reply:
Should be `utf8proc-2` instead, right? (I hate CMake's `find_` stuff, so take that with a grain of salt).

Reply:
Oups :/ Indeed. Cutiand pasting CMhinese is hard...
(It was still linked properly by libcrengine.so, as ldd showed.)

Reply:
oops, lol, thanks for close reading ;-)